### ISSUE TITLE
Randomise wording of release posts

### ISSUE THREAD

**Issue by jamesmbaazam** [epiverse-trace/epiverse-trace.github.io#132]
I think the release posts template stands a high chance of becoming monotonous in wording. I would suggest we randomize certain words in new posts to add a bit of dynamism in wording. 

For example, in https://github.com/epiverse-trace/epiverse-trace.github.io/blob/4421f498ad4af6ca5510b67df27450f014f50bbf/_templates/release.qmd#L9, we could sample synonyms for "excited" from a package like [syn](https://github.com/njtierney/syn). For example

```r
replicate(3, paste("We are", sample(as.vector(unlist((syn::syns("excited")))), 1), "to announce the release of a new"))
#> [1] "We are hopped up to announce the release of a new" 
#> [2] "We are breathless to announce the release of a new"
#> [3] "We are tickled to announce the release of a new"
```
Some of these are cringe but I'm sure we can find a way around this. This issue is just to sow a seed.

**Comment by jamesmbaazam** [comment]
Good point. I'll take a look in the course of the week.

**Comment by chartgerink** [comment]
@jamesmbaazam is this still something you are planning to work on? Given that you assigned it to yourself, I have held off on working on it - so I thought I'd check in 😊

**Comment by jamesmbaazam** [comment]
Hey Chris, I forgot to provide an update on this. I realised that the tidyverse was doing it manually and I figured it was for a good reason; the release posts are independent of the randomisation of words so if we're to go by this, we'll have to couple them somehow. I have therefore held back on implementing this.

Please feel free to close this issue or implement a solution that you deem fit.

**Comment by chartgerink** [comment]
Thanks @jamesmbaazam - I marked the issue as unplanned but will leave it open for others to work on if they wish. If this issue is not tackled over the next few months, we can close it at that point (for convenience: when the issue is a year old).

**Comment by Bisaloo** [comment]
> Hey Chris, I forgot to provide an update on this. I realised that the tidyverse was doing it manually and I figured it was for a good reason; the release posts are independent of the randomisation of words so if we're to go by this, we'll have to couple them somehow. I have therefore held back on implementing this. A manual solution would be to have a list of words/phrases to use and then record which post was assigned.
> 
> Please feel free to close this issue or implement a solution that you deem fit.

If I understand correctly, you're highlighting the fact we don't want a new random word to be selected each time the post is re-rendered, is that correct?

If that's the issue, a solution could be to select the word in `_scripts/create_release_post.R` and add it to the `.qmd` document as part of the `knitr::knit_expand()` call. The phrasing would then be hardcoded in the `.qmd` document and not susceptible to further accidental change.

**Comment by jamesmbaazam** [comment]
> If I understand correctly, you're highlighting the fact we don't want a new random word to be selected each time the post is re-rendered, is that correct?

No, but that is also a consideration. Changes like typos, etc, should not lead to a new word/phrase being resampled in `_templates/release.qmd`.

Let me rephrase: At the time of composing a new release blog post, we want to use a random word/phrase that is not the same as the one used in previous posts OR previous release posts from the same package. However, the workflow that generates the new post does not know what word/phrase the previous post used. So, we will need a way to track this.

**Comment by Bisaloo** [comment]
Ok, got it!

I'm not too concerned about this. I have seen a handful of tidyverse posts using the same phrase and I don't think it's a big issue. It'll still be on average less repetitive than the current solution.
